web/src/routes/EventsPage.tsx

```diff
@@ -29,17 +28,13 @@ import { cmsApiBaseUrl } from '../constants/urls'
 import DateFormatterContext from '../contexts/DateFormatterContext'
 import useWindowDimensions from '../hooks/useWindowDimensions'
 import featuredImageToSrcSet from '../utils/featuredImageToSrcSet'
-import { RouteProps } from './index'
 
-type PropsType = CityRouteProps & RouteProps<typeof EVENTS_ROUTE>
-
-const EventsPage = ({ cityModel, match, location, languages }: PropsType): ReactElement => {
-  const { cityCode, languageCode, eventId } = match.params
-  const pathname = normalizePath(location.pathname)
-  const history = useHistory()
+const EventsPage = ({ cityModel, languages, pathname, languageCode, cityCode }: CityRouteProps): ReactElement => {
+  const { eventId } = useParams()
```

Comment:
why  is there no need for a wildcard here?

Reply:
Because eventId is just one path segment (e.g. `event-1234`) while categoryId may have multiple segments, e.g. `willkommen/wilkommen-in-augsburg/erste-schritte`